src/etc/config.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <design>
+            <invalid_caches>
+                <full_page/>
```

Comment:
curious: what is this?

Reply:
It triggers a invalidation in full_page when design-config values are changed. When I saw it I thought it worked the other way around.
It comes from the page-cache module's config.xml file, but no need to duplicate it here. so, removed.